action.yml

```diff
@@ -145,8 +145,8 @@ runs:
         IMAGE_NAME: ${{ inputs.image_name}}
         REGISTRY_NAMESPACE: ${{ inputs.registry_namespace }}
         README_PATH: ${{ inputs.readme }}
-      run: update_quay_description.py
-      shell: python
+      run: python update_quay_description.py
```

Comment:
Is it really `python` or `python3`? How do you check it? Why not calling it over @setup_python GA?

Reply:
Ubuntu 20.04 contains by default python3 under 'python' command.
I did not use setup python action because the python in github runners is installed by default, and there is no need for specific version of python3.

Reply:
I don't like this approach, but let's not block it. And merge it.

Reply:
Agree, I do not like it either. But after few years working with GitHub Actions I can say these things just happen sometimes :).